settings.gradle

```diff
@@ -134,6 +134,10 @@ project(":libs").children.each { libsProject ->
   libsProject.name = "opensearch-${libsProject.name}"
 }
 
+project(":sandbox:libs").children.each { libsProject ->
```

Comment:
Why do this?

Reply:
This allows us to refer to the library as `opensearch-authn` similar to the other libraries in opensearch. This was only being applied to the libraries in `libs` so I extended it to apply to sandbox as well.